### PR TITLE
Bump to version 9.4

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
* https://github.com/facelessuser/pymdown-extensions/blob/9.4/requirements/project.txt
* https://github.com/facelessuser/pymdown-extensions/blob/9.4/pyproject.toml#L12

**Comment by dhirschfeld** [comment]
```
RuntimeError: Could not download https://pypi.io/packages/source/p/pymdown-extensions/pymdown-extensions-9.4.tar.gz
```
https://pypi.io/packages/source/p/pymdown-extensions/pymdown-extensions-9.4.tar.gz

**Comment by dhirschfeld** [comment]
https://files.pythonhosted.org/packages/62/78/d8390279ea0f9c627b2594497cb5d969e37ac81a3d0fd14803a856832788/pymdown_extensions-9.4.tar.gz

does seem to work

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,10 +21,11 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.7
+    - hatchling >=0.21.1
```
Comment (dhirschfeld) [inline recipe/meta.yaml:25]: https://github.com/facelessuser/pymdown-extensions/blob/9.4/pyproject.toml#L3

**Comment by dhirschfeld** [comment]
Will merge in the morning if no one beats me to it...

**Review by dhirschfeld — COMMENTED** [review]
:shipit:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: a80553b243d3ed2d6c27723bcd64ca9887e560e6f4808baa96f36e93061eaf90
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # FIXME:
+  # The above url did not work for version 9.4.
+  # If it works in future it should be used in preference to the hard-coded url below.
+  url: https://files.pythonhosted.org/packages/62/78/d8390279ea0f9c627b2594497cb5d969e37ac81a3d0fd14803a856832788/pymdown_extensions-9.4.tar.gz
```
[the remark this answers is not in the document]
Comment (ewels) [reply recipe/meta.yaml:13]: eg. Something like:
```suggestion
  url: https://github.com/facelessuser/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```

I _think_ that this should then still work with the automated conda-forge updates PRs (though I could be wrong about that).

**Comment by ewels** [comment]
Thanks @dhirschfeld !

**Pull request merged**